extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -212,14 +213,20 @@ export class InstrumentationService {
    * @private
    */
   createVisibilityListener_(callback, config) {
-    if (config['visibilitySpec']) {
+    const spec = config['visibilitySpec'];
+    if (spec) {
       if (!isVisibilitySpecValid(config)) {
         return;
       }
-
+      const self = this;
```

Comment:
I don't think you need this. See https://googlechrome.github.io/samples/arrows-es6/